README.md

```diff
@@ -6,21 +6,21 @@ See [Feature #12275](https://bugs.ruby-lang.org/issues/12275) for details.
 ## Usage
 
 ```ruby
-require 'string-undump'
+require 'string_undump'
```

Comment:
Yes, the name of this library is renamed from `string-undump` (hyphen) to `string_undump` (underscore) due to naming convention of C extension.